### PR TITLE
Update CMakeLists.txt to be more flexible

### PR DESCRIPTION
**Pull request by marcelotrevisani**
Update CMakeLists.txt to be more flexible to find the third parties libraries

Thanks for contributing to TVM!   Please refer to guideline https://docs.tvm.ai/contribute/ for useful information and tips. After the pull request is submitted, please request code reviews from [Reviewers](https://github.com/dmlc/tvm/blob/master/CONTRIBUTORS.md#reviewers).

### CODE REVIEW

**Comment by marcelotrevisani** [comment]
@junrushao1994 could you please take a look on that?

**Comment by junrushao1994** [comment]
Hey, this change is absolutely correct. Just to confirm, is this used for building conda-forge packages?

**Comment by marcelotrevisani** [comment]
Hey @junrushao1994 , yes, it would be good to have it, otherwise, I will have to patch the CMakeLists.txt, usually, in conda-forge we try to avoid third paties packages which are shipped along with the project. As I mentioned before, I already packaged those dependencies which tvm needs.

File: CMakeLists.txt [hunk 1]
```diff
@@ -51,11 +57,12 @@ tvm_option(USE_RANDOM "Build with random support" OFF)
...
+include_directories(${CMAKE_INCLUDE_PATH})
 include_directories("include")
-include_directories("3rdparty/dlpack/include")
-include_directories("3rdparty/dmlc-core/include")
-include_directories("3rdparty/rang/include")
-include_directories("3rdparty/compiler-rt")
+include_directories(${DLPACK_PATH})
+include_directories(${DMLC_PATH})
+include_directories(${RANG_PATH})
+include_directories(${RT_PATH})
```
Comment (junrushao1994) [inline CMakeLists.txt]: Let’s change it to COMPILER_RT_PATH

**Comment by junrushao1994** [comment]
BTW, what’s the status of current tvm/conda directory? Isn’t that suppose to be working properly with conda-forge?

**Comment by marcelotrevisani** [comment]
> BTW, what’s the status of current tvm/conda directory (https://github.com/dmlc/tvm/tree/master/conda)? Isn’t that suppose to be working properly with conda-forge?

Not with conda-forge, but it might help, because conda-forge has their own structure to build the package, usually the users just have to write a recipe and our infrastructure should do the rest

File: CMakeLists.txt (hunk 1 above)
Comment (marcelotrevisani) [reply CMakeLists.txt]: Sure, I am going to change it

**Comment by tqchen** [comment]
Thanks ,@marcelotrevisani @junrushao1994 this PR is now merged

**Pull request merged**